terraform/ec2/main.tf

```diff
@@ -405,7 +405,7 @@ resource "null_resource" "setup_sample_app_and_mock_server" {
   provisioner "remote-exec" {
     inline = [
       "sudo yum update -y",
-      "sudo amazon-linux-extras install docker -y",
+      "sudo amazon-linux-extra install docker -y",
```

Comment:
So the original command was wrong? Or are they different?

Reply:
oops, fixed it